portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,17 +34,21 @@
 @WebServlet("/data")
```

Comment:
Avoid these kind of changes even if minimal, they confuse the reviewer if the title only specifies frontend changes.

Reply:
Solved